README.md

```diff
@@ -48,15 +120,13 @@ Well played is a video game social media review app where users can add their fr
   - Game night
     - Might have trouble recreating a full outer join in a noSQL database
 
----
-
-### Group members and Work Breakdown
+## Group members and Work Breakdown
 
 - Justin Aitken - Flex dev
 - Julio Uribe - Frontend Specialist
 - Weyman Leung - Backend Specialist
```

Comment:
Can we update these so they're all flex? lol

Reply:
The whole commented out part is the pre proposal draft. I can make all of us flex as well.